src/test/java/uk/gov/hmcts/juror/api/moj/service/JurorManagementServiceImplTest.java

```diff
@@ -510,8 +503,6 @@ public void test_transferPoolMembers_happyPath_onePoolMember_firstTransfer() {
             .saveAndFlush(any());
         verify(jurorHistoryRepository, times(jurorNumbers.size()))
             .save(any());
-        verify(certLetterService, times(jurorNumbers.size()))
-            .enqueueNewLetter(any(), any());
```

Comment:
Should verify print service is called (Same for all)

Reply:
we dont need this as its a cert letter, these are not queued after discussion with Chris W - courts can separately print them off